packages/ui/react/src/components/Buttons/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Button } from '../Button'
+import { ButtonGroup } from './ButtonGroup'
+
```

Comment:
Hi @misaelucas could you add a few screenshots to the PR description showing how the component stories are displayed in Storybook when you run it locally?

That would help us have a perception of the visual look and feel of this component before merging the PR.